syntax_and_semantics/literals/regex.md

```diff
@@ -1,35 +1,67 @@
-# Regex
+# Regular Expressions
 
-Regular expressions are represented by the [Regex](http://crystal-lang.org/api/Regex.html) class, which is usually created with a literal:
+Regular expressions are represented by the [Regex](http://crystal-lang.org/api/Regex.html) class.
+
+A Regex is typically created with a regex literal using [PCRE](http://pcre.org/pcre.txt) syntax. It consists of a string of UTF-8 character enclosed in forward slashes (`/`):
 
 ```crystal
-foo_or_bar = /foo|bar/
-heeello    = /h(e+)llo/
-integer    = /\d+/
+/foo|bar/
+/h(e+)llo/
+/\d+/
+/ 
 ```
 
-A regular expression literal is delimited by `/` and uses [PCRE](http://pcre.org/pcre.txt) syntax.
-
-It can be followed by these modifiers:
-
-* i: ignore case (PCRE_CASELESS)
-* m: multiline (PCRE_MULTILINE)
-* x: extended (PCRE_EXTENDED)
+## Escaping
 
-For example
+Regular expressions support the same [escape sequences as String literals](./string.html).
 
 ```crystal
-r = /foo/imx
+/\// # slash
+/\\/ # backslash
+/\b/ # backspace
+/\e/ # escape
+/\f/ # form feed
+/\n/ # newline
+/\r/ # carriage return
+/\t/ # tab
+/\v/ # vertical tab
+/\NNN/ # octal ASCII character
+/\xNN/ # hexadecimal ASCII character
+/\uNNNN/ # hexadecimal unicode character
```

Comment:
Note: don't merge (or remove this) until https://github.com/crystal-lang/crystal/issues/5412 is fixed